src/components/BaseButton.vue

```diff
@@ -0,0 +1,4 @@
+<template>
+  <slot></slot>
```

Comment:
Please use `<button>` html tag here and add some attributes to it so you can easily change the button attributes.
```
Example: 

- Props (label, disabled, text color, background color, button text)
- Emits (click)
```